Libraries/Components/ScrollView/RecyclerViewBackedScrollView.android.js

```diff
@@ -113,6 +113,18 @@ var RecyclerViewBackedScrollView = React.createClass({
       );
     });
 
+    const refreshControl = this.props.refreshControl;
+    if (refreshControl) {
+      // Wrap the NativeAndroidRecyclerView with a AndroidSwipeRefreshLayout.
+      return React.cloneElement(
+        refreshControl,
+        {style: props.style},
+        <NativeAndroidRecyclerView {...props}>
```

Comment:
Can you add a style with `flex: 1` to `NativeAndroidRecyclerView` to make sure it fills the `AndroidSwipeRefreshLayout`.